tasks/main.yml

```diff
@@ -40,7 +40,7 @@
     virtualenv_python: python3.5
 
 - name: Run mr-provisioner setup.py install
-  shell: "cd {{mr_provisioner_build_path}}/mr-provisioner-{{mr_provisioner_version}} && {{mr_provisioner_venv_path}}/bin/python setup.py install"
+  shell: "{{mr_provisioner_venv_path}}/bin/python -m pip install pip==9.0.1 && cd {{mr_provisioner_build_path}}/mr-provisioner-{{mr_provisioner_version}} && {{mr_provisioner_venv_path}}/bin/python setup.py install"
 
```

Comment:
What about adding a new variable mr_provisioner_pip_version and make this configurable? if not mr_provisioner_pip_version .. ?